src/subcommand/wallet/burn.rs

```diff
@@ -0,0 +1,31 @@
+use {super::*, crate::outgoing::Outgoing, bitcoin::{opcodes}};
+
+#[derive(Debug, Parser)]
+pub struct Burn {
+  #[arg(long, help = "Don't sign or broadcast transaction")]
+  pub(crate) dry_run: bool,
+  #[arg(long, help = "Use fee rate of <FEE_RATE> sats/vB")]
+  fee_rate: FeeRate,
+  #[arg(
+    long,
+    help = "Target <AMOUNT> postage with sent inscriptions. [default: 10000 sat]"
+  )]
+  pub(crate) postage: Option<Amount>,
+  outgoing: Outgoing,
```

Comment:
```suggestion
  inscription_id: InscriptionId,
```
For now we only want to enable burning inscriptions, so let's make sure only that is possible. We'll add burning functionality to the other types (sats, btc, runes) later.